packages/fluentui/accessibility/src/behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -22,6 +25,7 @@ const treeTitleBehavior: Accessibility<TreeTitleBehaviorProps> = props => ({
         'aria-setsize': props.treeSize,
         'aria-posinset': props.index,
         'aria-level': props.level,
+        'aria-selected': props.selectable ? props.selected : undefined,
```

Comment:
```suggestion
        'aria-selected': props.selected : undefined,
```

Should it be simplified? As I understand `TreeTitle` should not be ever `selected` until it `Tree` will be `selectable`